be/src/olap/rowset/segment_v2/inverted_index_file_writer.cpp

```diff
@@ -496,7 +496,7 @@ InvertedIndexFileWriter::create_output_stream_v2() {
 
 void InvertedIndexFileWriter::write_version_and_indices_count(lucene::store::IndexOutput* output) {
     // Write the version number
-    output->writeInt(InvertedIndexStorageFormatPB::V2);
+    output->writeInt(_storage_format);
```

Comment:
may be we can change function name from create_output_stream_v2 to create_output_stream now